### DISCUSSION TITLE
How cephfs csi implements RWO

### DISCUSSION THREAD

**Discussion by band-p** [ceph/ceph-csi#3105] [Q&A] [answered]
I'm a newbie using cephfs csi, I don't know how to implement RWO in cephfs csi, can someone help me

**Comment by humblec** [comment, marked as answer]
@sys-liqian in PVC yaml file specify `accessmode` to `ReadWriteOnce`

**Reply by band-p** [reply]
thanks for the recovery, I would like to know the rationale, according to my understanding, cephfs should be mountable by every node

**Reply by Madhu-1** [reply]
RWO check is done at the Kubernetes level. AFAIK we don't have any check at the cephfs node plugin for this one. cephfs supports both RWO and RWX
